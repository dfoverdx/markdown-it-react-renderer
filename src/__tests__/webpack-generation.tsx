import React from 'react';
import { shallow } from 'enzyme';
import ReactifyMarkdown from '../../dist';

test('It loads the ReactifyMarkdown export', () => {
    const genComponent = () => shallow(<ReactifyMarkdown>Test</ReactifyMarkdown>);
    expect(genComponent).not.toThrow();
});