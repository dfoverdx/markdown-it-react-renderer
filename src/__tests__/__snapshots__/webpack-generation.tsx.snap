// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It loads the ReactifyMarkdown export 1`] = `
<p
  key="0"
>
  Test
</p>
`;
